ts/src/hyperliquid.ts

```diff
@@ -570,7 +570,7 @@ export default class hyperliquid extends Exchange {
         //     }
         //
         return [
-            this.safeInteger (ohlcv, 'T'),
+            this.safeInteger (ohlcv, 't'),
```

Comment:
@samgermain I'm not sure this is correct 
![image](https://github.com/ccxt/ccxt/assets/43336371/72558ba4-bec8-435a-9b95-c9bb89975a91)

the opening time of the candle can be different from the first trade time I think

Reply:
I separated the binance changes into a separate commit because of this delay https://github.com/ccxt/ccxt/pull/21811

Reply:
@carlosmiei I think that's just bad documentation on their part, the timestamps fall exactly on the minute time periods and are evenly spaced

```
% date -r 1710552360   
Fri Mar 15 21:26:00 EDT 2024
% date -r 1710552420   
Fri Mar 15 21:27:00 EDT 2024
% date -r 1710582180    
Sat Mar 16 05:43:00 EDT 2024
% date -r 1710582240    
Sat Mar 16 05:44:00 EDT 2024
% date -r 1710582300   
Sat Mar 16 05:45:00 EDT 2024
```

Reply:
@samgermain yeah just checked that, will merge it soon just waiting for the build (I suspect some response tests might break with this)